Sources/ArcGISToolkit/Components/AR/FlyoverSceneView.swift

```diff
@@ -70,16 +82,29 @@ public struct FlyoverSceneView: View {
     /// be effectively viewed in augmented reality. One such property is the camera controller.
     public init(
         initialLocation: Point,
+        initialHeading: Double? = nil,
         translationFactor: Double,
         @ViewBuilder sceneView: @escaping (SceneViewProxy) -> SceneView
```

Comment:
```suggestion
        translationFactor: Double,
        initialHeading: Double? = nil,
        @ViewBuilder sceneView: @escaping (SceneViewProxy) -> SceneView
```
Parameters with default arguments should be placed at the end of the parameter list.